packages/forms/src/widgets/DatalistWidget/DatalistWidget.js

```diff
@@ -207,7 +226,11 @@ class DatalistWidget extends React.Component {
 
 		const escapedValue = escapeRegexCharacters(this.getLabel(value).trim());
 		const regex = new RegExp(escapedValue, 'i');
-		return suggestions.filter(item => regex.test(this.getLabel(item)));
```

Comment:
Why did you removed `getLabel()` ?

Reply:
suggestions were list of 'value'(hero1, hero2, etc...), now suggestions are extended as list of objects({ label: 'Captain America', value: 'hero1' }, { label: 'Black Widow', value: 'hero2' }).
so here`getLabel(item)` here can be replaced with `item.label`;